packages/vega-util/src/extend.js

```diff
@@ -1,7 +0,0 @@
-export default function(_) {
-  for (let x, k, i=1, len=arguments.length; i<len; ++i) {
```

Comment:
In modern code, `arguments` is discouraged compared to using a `rest` parameter

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments#description